css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,402 @@
+		but is subject to any [=import conditions=]
+		(just as if declared by an ''@layer'' rule wrapped
+		in the appropriate [=conditional group rules=]).
+
+	* the optional <<import-conditions>>
+		states the [=import conditions=] under which it applies.
+
+	<div class=example>
+		A file with the <a href="https://www.w3.org/TR/css-cascade-5/#content-type">CSS Style Sheet Content-Type</a> 
+		and a URL of "foo.css":
+		<pre class=lang-css>
+		@sheet framework {
+			h1 { color: maroon; }
+		}
+		</pre>
+		A document with the following CSS and HTML:
+		<pre class=lang-css>
+		&lt;style>
+		@import framework from url(foo.css);
+		&lt;/style>
+		&lt;h1>Sample text&lt;/h1>
+		</pre>
+		In this example, the ''h1'' tag has the maroon color applied.
+	</div>
+
+	<div class="example">
+		The following <a href="#conditional-import">conditional <css>@import</css> rule</a>
+		only loads the [=named stylesheet=] with a name of "foo" from "narrow.css" when the UA
+		<a href="https://www.w3.org/TR/css-conditional-3/#support-definition">supports</a> ''display: flex'',
+		and only applies the [=stylesheet=] on a <a href="https://www.w3.org/TR/CSS2/media.html#media-types">handheld</a> device
+		with a <a href="https://www.w3.org/TR/mediaqueries-4/#width">maximum viewport width</a> of 400px.
+
+		<pre>@import foo from url("narrow.css") supports(display: flex) handheld and (max-width: 400px);</pre>
+	</div>
+
+	<div class="example">
+		The following layer imports the [=named stylesheet=]s named "foo" and "bar" from "tabs.css" into
+		the ''framework.component'' layer, and an un-named layer, respectively:
+
+		<pre>
+		@import foo, bar from url("tabs.css") layer(framework.component);
+		@import foo, bar from url("override.css") layer;
+		</pre>
+	</div>
+
+	If a <url-string> is provided,
+	it must be interpreted as a <<url>> with the same value.
+
+	<div class="example">
+		The following lines are equivalent in meaning
+		and illustrate both ''@import'' syntaxes
+		(one with ''url()'' and one with a bare string):
+
+		<pre class='lang-css'>
+		@import "mystyle.css";
+		@import url("mystyle.css");
+		</pre>
+	</div>
+
+<!--
+ ██████   ██████   ██████   ███████  ██     ██
+██    ██ ██    ██ ██    ██ ██     ██ ███   ███
+██       ██       ██       ██     ██ ████ ████
+██        ██████   ██████  ██     ██ ██ ███ ██
+██             ██       ██ ██     ██ ██     ██
+██    ██ ██    ██ ██    ██ ██     ██ ██     ██
+ ██████   ██████   ██████   ███████  ██     ██
+-->
+
+<h2 id="sheet-apis">Sheet APIs</h2>
+
+<h3 id="the-csssheetblockrule-interface">
+The <code>CSSSheetBlockRule</code> interface</h3>
+
+	The {{CSSSheetBlockRule}} interface represents
+	the ''@sheet'' [[#at-ruledef-sheet|block rule]]:
+
+	<pre class='idl' export>
+	[Exposed=Window]
+	interface CSSSheetBlockRule : CSSGroupingRule {
+	  readonly attribute CSSOMString name;
+	};
+	</pre>
+
+	Its <dfn attribute for=CSSSheetBlockRule>name</dfn> attribute represents
+	the <<sheet-name>> declared by the at-rule itself.
```

Comment:
Can at-rule be linked in any way?

Reply:
I'd actually suggest linking to ''@sheet'' instead of "at-rule" to give a more specific link in the formatted spec.

Reply:
But in this case, I went with Kevin's suggestion to link specifically to the `@sheet` definition